### DISCUSSION TITLE
How to use zip.js with vue-create-app?

### DISCUSSION THREAD

**Discussion by huangminsky** [gildas-lormeau/zip.js#283] [Q&A] [answered]
import * as zip from "@zip.js/zip.js";

 error  in ./node_modules/@zip.js/zip.js/lib/core/codecs/codec-pool-worker.js

Module parse failed: Unexpected token (110:57)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
|       function getWorker(options = {}) {
>               return new Worker(new URL(workerData.scripts[0], import.meta.url), options);
|       }
| 

 @ ./node_modules/@zip.js/zip.js/lib/core/codecs/codec-pool.js 34:0-47 55:9-18 60:10-19 69:11-20
 @ ./node_modules/@zip.js/zip.js/index.js
 @ ./node_modules/cache-loader/dist/cjs.js??ref--13-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??vue-loader-options!./src/components/HelloWorld.vue?vue&type=script&lang=js&
 @ ./src/components/HelloWorld.vue?vue&type=script&lang=js&
 @ ./src/components/HelloWorld.vue
 @ ./node_modules/cache-loader/dist/cjs.js??ref--13-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??vue-loader-options!./src/App.vue?vue&type=script&lang=js&
 @ ./src/App.vue?vue&type=script&lang=js&
 @ ./src/App.vue
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://10.10.83.96:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.js

don't know how to do ??? what loader I need ?

**Comment by huangminsky** [comment, marked as answer]
It's vue app ,not a react app 
Is the same NPM package applicable ？

**Reply by gildas-lormeau** [reply]
The fix is the same, you must include the loader @open-wc/webpack-import-meta-loader in your project. They documented how to add a loader here: https://cli.vuejs.org/guide/webpack.html#adding-a-new-loader.

**Reply by huangminsky** [reply]
Thank you for saving me a lot of time
